conans/assets/templates/__init__.py

```diff
@@ -0,0 +1,11 @@
+
+from jinja2 import DictLoader
+
+import conans.assets.templates.search_table_html
+
+SEARCH_TABLE_HTML = 'output/search_table.html'
```

Comment:
The "output" a bit unnecessary? Every html will be an output?

I would like to see "binaries" in the name, SEARCH_BINARIES_TABLE or something like that, to differentiate of other possible searches.

Reply:
Something to talk about. I see generators using this pattern too... and sometimes the CLI as well 😅  This is the reason why I prefer to have a folder, maybe `output/search_table.html` is not the best name, we have time to think a better one (I would like it to be coupled to the command that uses it, now it is `search --table=xxxx.html`, but I agree to align it with the future of the CLI)